cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataHttpHandler.java

```diff
@@ -100,12 +100,6 @@
     this.lineageAdmin = lineageAdmin;
   }
 
-  @GET
-  @Path("/**/metadata/subparts")
-  public void getSubparts(HttpRequest request, HttpResponder responder) throws BadRequestException {
-    //TODO(Rohit): Get the sub keys here and return that. This is the new API which we will support for browsing
-  }
-
```

Comment:
if we remove this, then how can we ever discover custom entities? Say, how can we find what fields in a dataset have metadata?